### DISCUSSION TITLE
在? 6.1.0.1 都来了，为什么还在用 6.1.0.2

### DISCUSSION THREAD

**Discussion by yxc0915** [QuickShop-Community/QuickShop-Hikari#1551] [General]
虽然我知道这是一个已知bug，但是我看到这个提示真的没绷住
![QQ截图20240509115710](https://github.com/Ghost-chu/QuickShop-Hikari/assets/62410385/accbcd8c-da0a-4269-ae93-01889ba4cdf5)

**Comment by YuanYuanOwO** [comment]
确认下个版本修复了

**Reply by yxc0915** [reply]
这个修复是认真的吗.......
![QQ截图20240519000947](https://github.com/Ghost-chu/QuickShop-Hikari/assets/62410385/139f10c6-de78-4905-abea-7de98db86f2e)

**Reply by YuanYuanOwO** [reply]
问过了，暂时不管更新检查了，考虑禁用一下 quickshop.alerts 权限吧

**Reply by yxc0915** [reply]
ok
